docbook/src/main/doc/adoc/content/chapter-2-Quick_Start.adoc

```diff
@@ -1,26 +1,21 @@
 == Quick Start
 
 JBossWS uses WildFly as its target container. The following examples
-focus on web service deployments that leverage EJB3 service
+focus on web service features that leverage EJB3 service
 implementations and the JAX-WS programming models. For further
-information on POJO service implementations and advanced topics you need
-consult the <<Jakarta XML Web Services User Guide>>.
+information on POJO service implementations and advanced topics
+consult the https://docs.jboss.org/author/display/JBWS/JAX-WS%20User%20Guide.html[Web Services User Guide].
```

Comment:
``<<Jakarta XML Web Services User Guide>>`` can be automatically transformed to the correct chapter link. Please use this format for the internal chapter link.

Reply:
done